tests/Company/CompanyServiceTest.php

```diff
@@ -453,4 +507,14 @@ private function buildUserCompanyService(string $email = 'customer-3@world-compa
 
         return new CompanyService($apiClient);
     }
+
+    private function setCompanyMetadata(int $companyId): CompanyUpdateCommand
+    {
+        $company = (new CompanyUpdateCommand($companyId))
```

Comment:
pas besoin de variable intermédiaire :)
```suggestion
        return (new CompanyUpdateCommand($companyId))
```

Reply:
```suggestion
        return new CompanyUpdateCommand($companyId);
```
Je dirai même plus :p